src/components/app/MenuButtons.js

```diff
@@ -473,13 +473,16 @@ const ProfileSharing = ({
       predictUrl={predictUrl}
     />
     <ProfileDownloadButton profile={profile} rootRange={rootRange} />
+    <a href="/docs/" target="_blank" className="menuButtonsLink">
+      Docs
```

Comment:
Best practice for links that open new tabs is adding `…`.

In this specific case the `?` symbol might also have enough perceivable affordance to stand on its own without label, saving some space.